### ISSUE TITLE
webp routes do not resolve

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/stacks#1303]
https://stacks.stanford.edu/image/iiif/bb112zx3193/bb112zx3193_00_0001/full/151,/0/default.webp should resolve, but it returns a 404.  https://stacks.stanford.edu/image/iiif/bb112zx3193/bb112zx3193_00_0001/full/151,/0/default.jpg does resolve.

**Comment by cbeer** [comment]
I'm not sure we claim to support webp. I thought  level 2 compliance only requires jpg + png.

**Comment by jcoyne** [comment]
Yes, it appears that Cantaloupe may not support it presently: https://github.com/cantaloupe-project/cantaloupe/issues/470#issuecomment-827731518
